### DISCUSSION TITLE
[@types/node] 'streams/promises' pipeline method overloads accepting only WHATWG streams

### DISCUSSION THREAD

**Discussion by samuelhnrq** [DefinitelyTyped/DefinitelyTyped#65886] [Issues with a @types package]
I've been using a lot of the node JS's streams API lately and came across an issue with the stream's package typing.

According to the streams documentation ([here](https://nodejs.org/docs/latest-v20.x/api/all.html#all_stream_streampipelinestreams-options)), it can accept a var-args of async generators, node JS streams and WHATWG streams, but the var-args overload is only accepting WHATWG streams ([here](https://github.com/xfournet/DefinitelyTyped/blob/master/types/node/stream.d.ts#L1366))

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
@samuelhnrq Code example?
